@navikt/core/react/src/provider/theme/AkselTheme.tsx

```diff
@@ -0,0 +1,55 @@
+import cl from "clsx";
+import React, { forwardRef } from "react";
+import { createContext } from "../../util/create-context";
+
+type AkselThemeContext = {
+  appearance: "light" | "dark";
+};
+
+const [ThemeProvider, useAkselTheme] = createContext<AkselThemeContext>({
+  errorMessage: "useAkselThemeProvider must be within AkselThemeProvider",
+  hookName: "useAkselThemeProvider",
```

Comment:
We usually have "Context" as suffix in the hook name...
```suggestion
  errorMessage: "useAkselThemeContext must be used within AkselThemeProvider",
  hookName: "useAkselThemeContext",
```
...or no suffix:
```suggestion
  errorMessage: "useAkselTheme must be used within AkselThemeProvider",
  hookName: "useAkselTheme",
```